src/NLopt.jl

```diff
@@ -195,6 +195,25 @@ struct ForcedStop <: Exception end
 # cache current exception for forced stop
 nlopt_exception = nothing
 
+"""
+    SavedException{E,B} <: Exception
```

Comment:
FWIW this is basically `CapturedException` (which is exported from Base)

Reply:
Would be good to use `CapturedException` then.

Reply:
Okay, I can updated